packages/react-components/src/components/VideoEffects/PresetVideoEffectsItems.tsx

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { _VideoEffectsItem, _VideoEffectsItemProps } from './VideoEffectsItem';
+
+/**
+ * 'None' Video Effects Item.
+ *
+ * @internal
+ */
+export const _VideoEffectsItemNoBackground = (props: _VideoEffectsItemProps): JSX.Element => {
+  const derivedProps = deriveProps(props, {
+    iconName: 'VideoEffectsNone',
+    title: 'None',
+    tooltipContent: 'Remove Background',
```

Comment:
What if we don't have any effect enabled, would the remove background be right tooltip in that case?

Reply:
Let's ask AlexP to include tooltips in his designs